javaagent-tooling/src/test/java/io/opentelemetry/javaagent/tooling/config/ConfigurationPropertiesSupplierTest.java

```diff
@@ -28,9 +27,8 @@
 @ClearSystemProperty(key = ConfigurationFile.CONFIGURATION_FILE_PROPERTY)
 class ConfigurationPropertiesSupplierTest {
 
-  @BeforeAll
-  @AfterAll
-  static void cleanUp() {
+  @BeforeEach
+  void setUp() {
     GlobalOpenTelemetry.resetForTest();
     ConfigurationFile.resetForTest();
```

Comment:
Since config file is reset only at the beginning of this tests other tests that run after this one will see the modified configuration instead of the empty one they would see when run on their own.